code/datums/mind.dm

```diff
@@ -347,7 +347,7 @@ GLOBAL_LIST_EMPTY(personal_objective_minds)
 			to_chat(current, span_nicegreen("My [S.name] grows to [SSskills.level_names[known_skills[S]]]!"))
 			SEND_SIGNAL(current, COMSIG_SKILL_RANK_INCREASED, S, known_skills[S], old_level)
 			GLOB.azure_round_stats[STATS_SKILLS_LEARNED]++
-			S.skill_level_effect(src, known_skills[S])
+			S.skill_level_effect(known_skills[S], src)
```

Comment:
(this is the one that 100% works)

Reply:
![image](https://github.com/user-attachments/assets/1a671bfc-9922-4170-be3e-09c1fe5454e1)